lib/glific/wa_group/wa_message.ex

```diff
@@ -0,0 +1,131 @@
+defmodule Glific.WaGroup.WaMessage do
+  @moduledoc false
+  use Ecto.Schema
+
+  alias Glific.Enums.{MessageFlow, MessageStatus, MessageType}
+
+  alias Glific.{
+    Contacts.Contact,
+    Flows.MessageBroadcast,
+    Groups.WAGroup,
+    Messages.MessageMedia,
+    Partners.Organization,
+    WaGroup.WaMessage,
+    WAGroup.WAManagedPhone
+  }
+
+  import Ecto.Changeset
+
+  @type t() :: %__MODULE__{
+          __meta__: Ecto.Schema.Metadata.t(),
+          id: non_neg_integer | nil,
+          uuid: Ecto.UUID.t() | nil,
+          type: String.t() | atom() | nil,
+          flow: String.t() | nil,
+          label: String.t() | nil,
+          status: String.t() | nil,
+          bsp_status: String.t() | nil,
+          bsp_id: String.t() | nil,
+          errors: map() | nil,
+          message_number: integer(),
+          contact_id: non_neg_integer | nil,
+          contact: Contact.t() | Ecto.Association.NotLoaded.t() | nil,
+          wa_group_id: non_neg_integer | nil,
+          wa_group: WAGroup.t() | Ecto.Association.NotLoaded.t() | nil,
+          media_id: non_neg_integer | nil,
+          media: MessageMedia.t() | Ecto.Association.NotLoaded.t() | nil,
+          organization_id: non_neg_integer | nil,
+          organization: Organization.t() | Ecto.Association.NotLoaded.t() | nil,
+          body: String.t() | nil,
+          context_id: String.t() | nil,
+          context_message_id: non_neg_integer | nil,
+          context_message: WaMessage.t() | Ecto.Association.NotLoaded.t() | nil,
+          message_broadcast_id: non_neg_integer | nil,
+          message_broadcast: MessageBroadcast.t() | Ecto.Association.NotLoaded.t() | nil,
+          wa_managed_phone_id: non_neg_integer | nil,
+          wa_managed_phone: WAManagedPhone.t() | Ecto.Association.NotLoaded.t() | nil,
```

Comment:
reorganize these in a way that 
1. fields comes first
2. foreign key
3. timestamps at last